packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -49,7 +79,8 @@ class JWPlayer extends React.Component {
         onResume={this.handleResume}
         onPause={this.handlePause}
         onOneHundredPercent={this.handleEnded}
-        {...this.props.jwpOptions}
+        onError={this.handleError}
+        {...{...this.props.jwpOptions, file: this.state.fileUrl}}
```

Comment:
maybe you can spread first in props, and define the file prop=this.state.fileUrl